diff --git a/dist/index.js b/dist/index.js
index e4d53c745ce584d06515874d016fa245cb43e01e..2627d2cbe1970be55321ee4fdf303fb0a1b154a1 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -194,7 +194,7 @@ class GoogleGenerativeAIResponseError extends GoogleGenerativeAIError {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const BASE_URL = "https://generativelanguage.googleapis.com";
+const BASE_URL = process.env.G_API_URL || "https://generativelanguage.googleapis.com";
 const DEFAULT_API_VERSION = "v1";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
diff --git a/dist/index.mjs b/dist/index.mjs
index 71aaa0f85d6bb7aa0b3cf99fe911bcea5fc43411..3831019b56e10f8fb4690488e694a61b4b9530d7 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -192,7 +192,7 @@ class GoogleGenerativeAIResponseError extends GoogleGenerativeAIError {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const BASE_URL = "https://generativelanguage.googleapis.com";
+const BASE_URL = process.env.G_API_URL || "https://generativelanguage.googleapis.com";
 const DEFAULT_API_VERSION = "v1";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
