diff --git a/dist/index.js b/dist/index.js
index 8c3b18c94db5523b2f4c5baa7a6b25fb07c642af..ee8af65c1aaef90bcc6bf350bd77709b745b730a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -152,7 +152,7 @@ class GoogleGenerativeAIResponseError extends GoogleGenerativeAIError {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const BASE_URL = "https://generativelanguage.googleapis.com";
+const BASE_URL = process.env.G_API_URL || "https://generativelanguage.googleapis.com";
 const API_VERSION = "v1";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
diff --git a/dist/index.mjs b/dist/index.mjs
index 89630066aeb7b8f904702ff7b0e999538f1f2322..0d9a4d6cc3819f2149bbf44dbd57f78992a5664a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -150,7 +150,7 @@ class GoogleGenerativeAIResponseError extends GoogleGenerativeAIError {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const BASE_URL = "https://generativelanguage.googleapis.com";
+const BASE_URL = process.env.G_API_URL || "https://generativelanguage.googleapis.com";
 const API_VERSION = "v1";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
