diff --git a/lib/cjs/fetch.js b/lib/cjs/fetch.js
index ab40f1eeff0ec0a30043e45478f81cd1dc845adb..fd2a0f92f9692dcf47a6ae21baa77f5ac5e64c36 100644
--- a/lib/cjs/fetch.js
+++ b/lib/cjs/fetch.js
@@ -48,7 +48,7 @@ function fetchEventSource(input, _a) {
         const onopen = inputOnOpen !== null && inputOnOpen !== void 0 ? inputOnOpen : defaultOnOpen;
         async function create() {
             var _a;
-            curRequestController = new AbortController();
+            curRequestController = curRequestController || new AbortController();
             try {
                 const response = await fetch(input, Object.assign(Object.assign({}, rest), { headers, signal: curRequestController.signal }));
                 await onopen(response);
diff --git a/lib/esm/fetch.js b/lib/esm/fetch.js
index 31eb2278da53ba79d9fc78ea32d69f2d15f325ff..be5d2ccba704c59c7773a47ecfb084925c3032f0 100644
--- a/lib/esm/fetch.js
+++ b/lib/esm/fetch.js
@@ -45,7 +45,7 @@ export function fetchEventSource(input, _a) {
         const onopen = inputOnOpen !== null && inputOnOpen !== void 0 ? inputOnOpen : defaultOnOpen;
         async function create() {
             var _a;
-            curRequestController = new AbortController();
+            curRequestController = curRequestController || new AbortController();
             try {
                 const response = await fetch(input, Object.assign(Object.assign({}, rest), { headers, signal: curRequestController.signal }));
                 await onopen(response);
