diff --git a/dist/themes/css-variables.d.mts b/dist/themes/css-variables.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/css-variables.mjs b/dist/themes/css-variables.mjs
deleted file mode 100644
index 4ae50e7d938f75ecda09fe8211e883eff01642c0..0000000000000000000000000000000000000000
diff --git a/dist/themes/dark-plus.d.mts b/dist/themes/dark-plus.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/dark-plus.mjs b/dist/themes/dark-plus.mjs
deleted file mode 100644
index e1e8a3c40d37b51779d84043e454efa68a8c06ef..0000000000000000000000000000000000000000
diff --git a/dist/themes/dracula-soft.d.mts b/dist/themes/dracula-soft.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/dracula-soft.mjs b/dist/themes/dracula-soft.mjs
deleted file mode 100644
index 3f4d8a66ea059ef5eeba31036c7fbe8492f7f3b9..0000000000000000000000000000000000000000
diff --git a/dist/themes/dracula.d.mts b/dist/themes/dracula.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/dracula.mjs b/dist/themes/dracula.mjs
deleted file mode 100644
index bdbe344447491924f83b7df06ddbac70168af5e1..0000000000000000000000000000000000000000
diff --git a/dist/themes/github-dark-dimmed.d.mts b/dist/themes/github-dark-dimmed.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/github-dark-dimmed.mjs b/dist/themes/github-dark-dimmed.mjs
deleted file mode 100644
index ea4266e7f6f0d8be6c3717ea6a613aa7812911bb..0000000000000000000000000000000000000000
diff --git a/dist/themes/github-light.d.mts b/dist/themes/github-light.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/github-light.mjs b/dist/themes/github-light.mjs
deleted file mode 100644
index 0916918ab99ab119021de129d851f3fd2d88a367..0000000000000000000000000000000000000000
diff --git a/dist/themes/hc_light.d.mts b/dist/themes/hc_light.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/hc_light.mjs b/dist/themes/hc_light.mjs
deleted file mode 100644
index a172977ca7a91b5a0f99d84373c6d180fa88a708..0000000000000000000000000000000000000000
diff --git a/dist/themes/light-plus.d.mts b/dist/themes/light-plus.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/light-plus.mjs b/dist/themes/light-plus.mjs
deleted file mode 100644
index 72d599b18711dbc84c5b738504eef7a38f9d1e82..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-darker.d.mts b/dist/themes/material-theme-darker.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-darker.mjs b/dist/themes/material-theme-darker.mjs
deleted file mode 100644
index 14b2cb049014fb75689e4b497e85cfaad8b12b4a..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-lighter.d.mts b/dist/themes/material-theme-lighter.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-lighter.mjs b/dist/themes/material-theme-lighter.mjs
deleted file mode 100644
index f5d2b45f23934371308d55a9422f4510ee1b93c3..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-ocean.d.mts b/dist/themes/material-theme-ocean.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-ocean.mjs b/dist/themes/material-theme-ocean.mjs
deleted file mode 100644
index 560c8e0fad559db0b1fe10aafb06b1194f8363b3..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-palenight.d.mts b/dist/themes/material-theme-palenight.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme-palenight.mjs b/dist/themes/material-theme-palenight.mjs
deleted file mode 100644
index d53d0251ddd3ab66ca858c8945714fa8771112ab..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme.d.mts b/dist/themes/material-theme.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/material-theme.mjs b/dist/themes/material-theme.mjs
deleted file mode 100644
index afa02af6d880b04f4ebfd7b6ddbf51fdf97cc59c..0000000000000000000000000000000000000000
diff --git a/dist/themes/min-dark.d.mts b/dist/themes/min-dark.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/min-dark.mjs b/dist/themes/min-dark.mjs
deleted file mode 100644
index 67f4e1260ca7e7674f34f97a0ef4707f65cc8032..0000000000000000000000000000000000000000
diff --git a/dist/themes/min-light.d.mts b/dist/themes/min-light.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/min-light.mjs b/dist/themes/min-light.mjs
deleted file mode 100644
index 17d9311f7a8b9237a8d2d7d05b2d4a670cdf454d..0000000000000000000000000000000000000000
diff --git a/dist/themes/monokai.d.mts b/dist/themes/monokai.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/monokai.mjs b/dist/themes/monokai.mjs
deleted file mode 100644
index 192f2c8d7a82c4fd436a109e9c948355444160cd..0000000000000000000000000000000000000000
diff --git a/dist/themes/nord.d.mts b/dist/themes/nord.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/nord.mjs b/dist/themes/nord.mjs
deleted file mode 100644
index 0e5e51acf3c2d18cf1c812ec1a99df4544e6859d..0000000000000000000000000000000000000000
diff --git a/dist/themes/one-dark-pro.d.mts b/dist/themes/one-dark-pro.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/one-dark-pro.mjs b/dist/themes/one-dark-pro.mjs
deleted file mode 100644
index 168d24c84cf50d901ba639833bf0f0bfdf4677d4..0000000000000000000000000000000000000000
diff --git a/dist/themes/poimandres.d.mts b/dist/themes/poimandres.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/poimandres.mjs b/dist/themes/poimandres.mjs
deleted file mode 100644
index a36d9057b375739ebc918dc1ff6b11e277114ad2..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine-dawn.d.mts b/dist/themes/rose-pine-dawn.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine-dawn.mjs b/dist/themes/rose-pine-dawn.mjs
deleted file mode 100644
index 813424981104f82df4b59eaf2cb97d9827438ce3..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine-moon.d.mts b/dist/themes/rose-pine-moon.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine-moon.mjs b/dist/themes/rose-pine-moon.mjs
deleted file mode 100644
index 65911e4adee85b1693a32975e6cd78e613186b61..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine.d.mts b/dist/themes/rose-pine.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/rose-pine.mjs b/dist/themes/rose-pine.mjs
deleted file mode 100644
index d1c2fe7a9c45d285e4de0f0ed2e3bceeefc38f36..0000000000000000000000000000000000000000
diff --git a/dist/themes/slack-dark.d.mts b/dist/themes/slack-dark.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/slack-dark.mjs b/dist/themes/slack-dark.mjs
deleted file mode 100644
index aa03ca9b67c6074e9fe9f3d76a749c6be49165e3..0000000000000000000000000000000000000000
diff --git a/dist/themes/slack-ochin.d.mts b/dist/themes/slack-ochin.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/slack-ochin.mjs b/dist/themes/slack-ochin.mjs
deleted file mode 100644
index a0ca88ffe06077fdcb4d5ee213497489de4455ba..0000000000000000000000000000000000000000
diff --git a/dist/themes/solarized-dark.d.mts b/dist/themes/solarized-dark.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/solarized-dark.mjs b/dist/themes/solarized-dark.mjs
deleted file mode 100644
index 26653940bc3c80c0daedbc19f2889c1293fa74aa..0000000000000000000000000000000000000000
diff --git a/dist/themes/solarized-light.d.mts b/dist/themes/solarized-light.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/solarized-light.mjs b/dist/themes/solarized-light.mjs
deleted file mode 100644
index 02c969614044d6e24aa7f0e14751f029af5fefb7..0000000000000000000000000000000000000000
diff --git a/dist/themes/vitesse-dark.d.mts b/dist/themes/vitesse-dark.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/vitesse-dark.mjs b/dist/themes/vitesse-dark.mjs
deleted file mode 100644
index 07eed105f411072ff821017560ae662616eb87ae..0000000000000000000000000000000000000000
diff --git a/dist/themes/vitesse-light.d.mts b/dist/themes/vitesse-light.d.mts
deleted file mode 100644
index 04990a046926513b3a253e6d3094b5cb2eb7b700..0000000000000000000000000000000000000000
diff --git a/dist/themes/vitesse-light.mjs b/dist/themes/vitesse-light.mjs
deleted file mode 100644
index c535d678fad3227cb742c2f40b9e01278010456b..0000000000000000000000000000000000000000
diff --git a/dist/themes.mjs b/dist/themes.mjs
index 7493f01c31d673a0897da27315ee4146e739b803..d839f9d7a7b8a65cd4f7cb2e69c1d9b38109e85f 100644
--- a/dist/themes.mjs
+++ b/dist/themes.mjs
@@ -1,33 +1,5 @@
 const bundledThemes = {
-  "css-variables": () => import('./themes/css-variables.mjs'),
-  "dark-plus": () => import('./themes/dark-plus.mjs'),
-  "dracula": () => import('./themes/dracula.mjs'),
-  "dracula-soft": () => import('./themes/dracula-soft.mjs'),
   "github-dark": () => import('./themes/github-dark.mjs'),
-  "github-dark-dimmed": () => import('./themes/github-dark-dimmed.mjs'),
-  "github-light": () => import('./themes/github-light.mjs'),
-  "hc_light": () => import('./themes/hc_light.mjs'),
-  "light-plus": () => import('./themes/light-plus.mjs'),
-  "material-theme": () => import('./themes/material-theme.mjs'),
-  "material-theme-darker": () => import('./themes/material-theme-darker.mjs'),
-  "material-theme-lighter": () => import('./themes/material-theme-lighter.mjs'),
-  "material-theme-ocean": () => import('./themes/material-theme-ocean.mjs'),
-  "material-theme-palenight": () => import('./themes/material-theme-palenight.mjs'),
-  "min-dark": () => import('./themes/min-dark.mjs'),
-  "min-light": () => import('./themes/min-light.mjs'),
-  "monokai": () => import('./themes/monokai.mjs'),
-  "nord": () => import('./themes/nord.mjs'),
-  "one-dark-pro": () => import('./themes/one-dark-pro.mjs'),
-  "poimandres": () => import('./themes/poimandres.mjs'),
-  "rose-pine": () => import('./themes/rose-pine.mjs'),
-  "rose-pine-dawn": () => import('./themes/rose-pine-dawn.mjs'),
-  "rose-pine-moon": () => import('./themes/rose-pine-moon.mjs'),
-  "slack-dark": () => import('./themes/slack-dark.mjs'),
-  "slack-ochin": () => import('./themes/slack-ochin.mjs'),
-  "solarized-dark": () => import('./themes/solarized-dark.mjs'),
-  "solarized-light": () => import('./themes/solarized-light.mjs'),
-  "vitesse-dark": () => import('./themes/vitesse-dark.mjs'),
-  "vitesse-light": () => import('./themes/vitesse-light.mjs')
 };
 
 export { bundledThemes };
